src/main/java/net/countercraft/movecraft/warfare/MovecraftWarfare.java

```diff
@@ -28,18 +28,23 @@
     private static MovecraftWarfare instance;
     private AssaultManager assaultManager;
     private SiegeManager siegeManager;
+    private File configFile;
 
     public static synchronized MovecraftWarfare getInstance() {
         return instance;
     }
 
     @Override
     public void onEnable() {
+        configFile = new File(getDataFolder(), "config.yml");
         if(instance != null)
             return;
 
         instance = this;
 
+        final String packageName = getServer().getClass().getPackage().getName();
+        Config.IsLegacy = Integer.parseInt(packageName.substring(packageName.lastIndexOf(".") + 1).split("_")[1]) <= 12;
+
```

Comment:
I do not wish to add 1.13+ support at this time.

Reply:
To expand on this,
  I am working on Movecraft-Repair which will abstract out all WorldEdit functionality and will (eventually) have compatibility with both WE6 and WE7.  Similarly, the future plugin Movecraft-WorldGuard will bring all WorldGuard functionality of Movecraft and create an abstraction layer for plugins such as this.
  
Once these two addons are completed with PRs to both your fork and APDevTeam accepted, plugins such as Movecraft-Warfare and any other addons coming will transition to using Movecraft-Repair and Movecraft-Worldguard dependencies instead of internal WE6/WG6 functions, bringing full 1.13+ support quickly.  Movecraft-Combat's WG support will also be brought to Movecraft-Worldguard eventually...